script/Deployer.sol

```diff
@@ -57,7 +58,9 @@ contract Deployer is Script {
         investmentManager = new InvestmentManager(address(root), address(escrow));
         poolManager = new PoolManager(address(escrow), vaultFactory, restrictionManagerFactory, trancheTokenFactory);
 
-        centrifugeRouter = new CentrifugeRouter(address(poolManager));
+        routerEscrow = new Escrow(deployer);
```

Comment:
Custom escrow because
1. Cleaner to not mix funds with core contracts
2. Avoids the vault doing a transfer from escrow to escrow